tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -96,6 +96,70 @@ Management network - valid
     Run Regression Tests
     Cleanup VIC Appliance On Test Server
 
+Connectivity Bridge to Public
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    ${out}=  Run  govc host.portgroup.remove bridge
+    ${out}=  Run  govc host.portgroup.remove vm-network
+    ${out}=  Run  govc host.portgroup.remove management
+
+    Log To Console  Create a public portgroup 
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitch0 vm-network
+    Should Be Empty  ${out}
+
+    Log To Console  Create a bridge portgroup
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitch0 bridge
+    Should Be Empty  ${out}
+
+    Log To Console  Create a management portgroup.
+    ${out}=  Run  govc host.portgroup.add -vswitch vSwitch0 management
+    Should Be Empty  ${out}
+
+    ${output}=  Run  bin/vic-machine-linux create --debug 1 --name=%{VCH-NAME} --target=%{TEST_URL}%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD} --force=true --bridge-network=bridge --public-network=vm-network --compute-resource=%{TEST_RESOURCE} --container-network management --container-network vm-network --container-network-ip-range=management:10.10.10.0/24 --container-network-gateway=management:10.10.10.1/24 --no-tlsverify
```

Comment:
Not needed for this test, but testing the superset functionality on this so I can plug this in for the bridge to container ping test.